app/views/spree/shared/_registration_header.html.erb

```diff
@@ -0,0 +1,8 @@
+<nav class="breadcrumb-wrap d-flex flex-wrap align-items-center justify-content-between">
+  <ol class="breadcrumb">
+    <li class="breadcrumb-item"><a href="#">Home</a></li>
+    <li class="breadcrumb-item active">Registration</li>
+  </ol>
+</nav>
+<h1 class="title mb-0">Registration</h1>
```

Comment:
Remove this partial and move these two line to devise -> signUp form file.

<h1 class="title mb-0">Registration</h1>
<div class="sm-title mb-20">It looks like you’re new here. We need a bit more info...</div>